framework/Source/GPUImageMovie.m

```diff
@@ -695,6 +695,14 @@ - (void)endProcessing;
         [currentTarget endProcessing];
     }
     
+    // dgm: added this - otherwise we were getting crashes when the playerItemOutput
+    // was processing already scheduled work after it had supposedly been stopped.
+    if (playerItemOutput && self.playerItem) {
+        [self.playerItem removeOutput:playerItemOutput];
```

Comment:
I don't see anywhere it was currently being removed, which seemed to be the problem - seems safe to me.  Also nilling it out at this point should take care of problems of it happening twice accidentally @msfeldstein